scripts/setup-certs.mjs

```diff
@@ -0,0 +1,196 @@
+import axios from 'axios';
+import fs from 'node:fs/promises';
+import { EOL } from 'os';
+import { createRequire } from 'module';
+import { readJSON, scrubForFilename, waitForJob } from './helpers.mjs';
+
+const require = createRequire(import.meta.url);
+const profiles = require('./ecosystem-requests/clarity_partners_16/profiles/profiles.json');
+const ecosystem = require('./ecosystem-requests/clarity_partners_16/clarity_partners_16.json');
+
+const axiosHeaders = {
+  headers: {
+    'DOCK-API-TOKEN': `${process.env.DOCK_API_TOKEN}`,
+  },
+};
+
+const newEnvironment = {
+  DOCK_API_TOKEN: process.env.DOCK_API_TOKEN,
+  DOCK_API_URL: process.env.DOCK_API_URL,
+  NEXT_PUBLIC_DOCK_API_URL: process.env.DOCK_API_URL
+};
+
+async function writeEnvFile() {
+  console.log('--- Writing out env vars to env.new file ---');
+  const envFile = await fs.open('.env.new', 'a');
+  try {
+    const lines = await Object.keys(newEnvironment).map(async (key) => {
+      console.log(`\t${key}`);
+      await envFile.appendFile(`${key}=\"${newEnvironment[key]}\"${EOL}`);
+    });
+
+    await Promise.all(lines);
+  } catch (error) {
+    console.error('error writing env file', error);
+  } finally {
+    await envFile.close();
+  }
+}
+
+async function createProfiles() {
+  const didsUrl = `${process.env.DOCK_API_URL}/dids`;
+  const profilesUrl = `${process.env.DOCK_API_URL}/profiles`;
+
+  const participantProfiles = {};
+  const proofRequestTemplates = [];
+
+  let convenerDID;
+  console.log('--- Creating profiles ---');
+
+  const requests = await profiles.map(async (profile) => {
+    console.log(`\t ${profile.name}`);
+
+    try {
+      const didResponse = await axios.post(didsUrl, {}, axiosHeaders);
+
+      await waitForJob(didResponse.data.id, axiosHeaders);
+      profile.did = didResponse.data.did;
+      const profileResponse = await axios.post(profilesUrl, profile, axiosHeaders);
+      if (profile.isParticipant) {
+        participantProfiles[scrubForFilename(profile.name)] = profileResponse.data;
+      }
+
+      if (profile.isIssuer) {
+          newEnvironment.DOCK_API_DID = profileResponse.data.did;
+          convenerDID = profileResponse.data.did;
+
+          if (profile.envVar) {
+            newEnvironment[profile.envVar] = profileResponse.data.did;
+          }
+      }
+
+      if (profile.proofRequests) {
+        const allRequests = await profile.proofRequests.map(async (proofRequest) => {
```

Comment:
`const allRequests = await profile.proofRequests.map` should be `const allRequests = profile.proofRequests.map`